Codeinwp/neve
# Fix post type enhancements meta defaults

### Summary
- The default value was not in the required format so it was ignored

### Will affect the visual aspect of the product
NO

### Test instructions
- Install Neve Pro and this version of Neve
- Install Custom Post Types UI, and create a custom post type.
- Create a post for the previous post type.
- Go to Appearance > Customize > Layout > Single movie ( in my example )
- Enable the Use of a custom layout option
- Post meta should appear from the beginning


Closes #3537.
